src/ecrc-frontend/src/components/base/menuItem/MenuItem.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { create } from "react-test-renderer";
+
+import MenuItem from "./MenuItem";
+
+describe("MenuItem Component", () => {
+  test("Matches the snapshot", () => {
+    const link = {
+      url: "/somewhere",
+      name: "Link to Somewhere"
+    };
+
+    const menuItem = create(<MenuItem name={link.name} url={link.url} />);
```

Comment:
shouldnt this be something like this after the change you made? change `link` to `menuItem` and it should be good
```suggestion
    const menuItem = {};
    const menuItem = create(<MenuItem menuItem={menuItem} />);
```

Reply:
Fixed!